src/components/Form/ServerPaginatedDDList.js

```diff
@@ -55,14 +55,23 @@ const ServerPaginatedDDList = (props) => {
         getUrlParams,
         serverListClassName,
         ddItemHeight,
+        minPageNo,
         ...restProps
     } = props;
     const [ items, setItems ] = useState([]);
     const [ itemsResetCounter, setItemsResetCounter ] = useState(0);
     const [ total, setTotal ] = useState(0);
     const [ hasNextPage, setHasNextPage ] = useState(false);
     const [ isNextPageLoading, setIsNextPageLoading ] = useState(true);
-    const pageNoRef = useRef(1);
+
+    const getPageNoRef = () => {
+        if (minPageNo === 0) {
```

Comment:
This check would take minPageNo only if it is 0, but as a generic config, if any consumer of this component provides `minPageNo` it has to take its value regardless of if its 0 or not.

```
return minPageNo ?? 1
```
The above line would return minPageNo if it is not null or undefined otherwise 1